### DISCUSSION TITLE
zelzele.live

### DISCUSSION THREAD

**Discussion by LatifY** [acikkaynak/deprem-yardim-projesi#42] [Show and tell]
https://www.zelzele.live

<img src="https://www.zelzele.live/banner.png" width="50%" heigth="50%">

Minik dostlarımız için çok fazla web sitesi yoktu. Ben de hem onlar için kayıp ihbarı yapabileceğiniz hem de yararlı bazı servislere ulaşabileceğimiz küçük bir platform hazırladım. Kayıp ihbarları yakinimibul.net üzerinden geliyor. Böylece daha fazla kişiye ulaşılabileceğini düşündüm...

**Comment by sercangoger** [comment]
deprem.io yararlı linkler bölümüne eklenebilir.

**Reply by LatifY** [reply]
ekli zaten 3. sırada bulunuyor deprem.io

**Reply by eece** [reply]
Tam tersini söylemiş aslında, deprem.io'dan zelzele.live'a link verilmesi yani.

**Reply by LatifY** [reply]
tamamdır şimdi anladım :) ne yazık ki yeni proje almıyolarmış üzüldüm ama yapacak bir şey yok

**Comment by eece** [comment]
Gayet sade ve anlaşılır olmuş elinize sağlık.

- Listeleme kısmına şehir filtresi ekleyebilirsiniz. 
- Bildirim formunun başka sitede host edilmesi kafa karıştırıyor.

**Reply by LatifY** [reply]
artık sitede bildiri formu da bulunuyor. filtreler üzerinde de çalışıyorum

**Reply by eece** [reply]
Süper :) Filtre için de firebase'in query'sinde bir filter methodu olabilir ya da dönen datayı js tarafında filtreleyebilirsiniz.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Latif ile iletişime geçilerek süreç başlatılmış.
